.github/workflows/deploy-branch.yml

```diff
@@ -0,0 +1,46 @@
+name: Deploy-dev
```

Comment:
let's name it `deploy-branch`, just like in the filename. 
Otherwise the page https://github.com/tarantool/doc/actions/workflows/deploy-branch.yml is a little confusing.

Reply:
Fixed

Reply:
Is it?